neuralop/models/fno.py

```diff
@@ -4,7 +4,13 @@
 import torch.nn.functional as F
 
 from ..layers.spectral_convolution import SpectralConv
-from ..layers.spherical_convolution import SphericalConv
+# Only reference SphericalConv if torch_harmonics is built locally
```

Comment:
What about just moving the whole sfno logic to an sfno file?

Reply:
Sounds good. done